packages/styles/src/elements/anchor.scss

```
@@ -0,0 +1,38 @@
+@use '../functions/tokens';
+@use '../tokens/elements';
+@use '../mixins/utilities';
+
+tokens.$default-map: elements.$post-link;
+
+a {
+  text-decoration: tokens.get('link-decoration');
+  border-radius: tokens.get('link-border-radius');
+  color: inherit;
+
+  &:hover {
+    color: tokens.get('link-hover-fg');
+  }
+
+  @include utilities.focus-style(
+    '',
+    tokens.get('link-focus-outline-offset'),
+    tokens.get('link-focus-outline-width'),
+    tokens.get('link-focus-outline-color'),
+    tokens.get('link-focus-outline-style')
+  );
```

It's a detail but I have seen Tim write the focus styles like so and it make it more readable. What do you think? 

```suggestion
  @include utilities.focus-style(
    $offset: tokens.get('link-focus-outline-offset'),
    $width: tokens.get('link-focus-outline-width'),
    $color: tokens.get('link-focus-outline-color'),
    $style: tokens.get('link-focus-outline-style'),
  );
```